src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,112 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static User newUser;
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        newUser = new User();
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @AfterAll
+    static void afterAll() {
+        Storage.people.clear();
+    }
+
+    @BeforeEach
+    void setUp() {
+        newUser.setAge(19);
+        newUser.setLogin("Djon");
+        newUser.setPassword("qwerty123");
+    }
+
+    @Test
+    void nullValue_NotOk() {
```

Comment:
![Screenshot 2022-05-24 at 15 25 39](https://user-images.githubusercontent.com/60258373/170034188-c65e839f-f6fa-45c3-ae91-c07212220e30.png)
 the same for other methods